### DISCUSSION TITLE
use pinia-plugin-persistedstate with pinia-orm

### DISCUSSION THREAD

**Discussion by unknown1337** [CodeDredd/pinia-orm#202] [Q&A] [answered]
Dear,

I'm starting to get a bit used to pinia-ORM, thanks!!

Now I would like my stores to be persistent, how can I use the pinia-plugin-persistedstate? I installed the plugin, registered it on the pinia store (.use() ). Though I don't know how to put the persist=true in my 'Model' / store? 

Example that I've tried:
1) .... extends Model{ .... get persist(){return true}}
2) .... extends Model{ .... static persist = true}

What works: I can put data in my pinia-orm stores and see the content in the vuedevtools.
What does not work: when I refresh the page (quasar webapp) I see in the vue devtools that the same stores that were pre-filled are empty :(

THANKS!

**Comment by CodeDredd** [comment, marked as answer]
Look at https://pinia-orm.codedredd.de/guide/model/pinia-options#extending-the-store-options-eg-for-plugins

**Reply by unknown1337** [reply]
Thanks, also for the fast reply!! That worked!. 

(note, there is a small typo in the text: "to pass **otpions** to your store "  otpions->options.

**Reply by CodeDredd** [reply]
Thanks for the notice 👍
